test/binary-parser.test.js

```diff
@@ -371,7 +371,7 @@ function pathSetBinaryTests() {
         currency: "BTC",
         issuer: "r3AWbdp2jQLXLywJypdoNwVSvr81xs3uhn",
       },
-      { currency: "0000000000000000000000005852500000000000" },
+      { currency: "XRP" },
```

Comment:
Are we be expecting this case to throw?

Reply:
Yeah I changed it from dodgy XRP to a correct implementation of XRP. The code treats this as all 0s, as it should.